doc/user/content/sql/insert.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "INSERT"
+description: "`INSERT` inserts values into a table."
+menu:
+  main:
+    parent: 'sql'
+---
+
+`INSERT` inserts values into a table.
+
+## Conceptual framework
+
+`INSERT` statements inserting data into [tables](../create-table).
+
+## Syntax
+
+{{< diagram "insert.svg" >}}
+
+Field | Use
+------|-----
+_table&lowbar;name_ | The name of the target table.
+_col&lowbar;value_ | The value to be inserted into the column, or `NULL` if column is nullable.
```

Comment:
This is very pedantic, but the very literal reading of this makes it sound like nullable columns can only accept null values. I'd make the bit about `NULL` a second sentence to help make it unambiguous.